asllib/menhir2bnfc/README.md

```diff
@@ -86,6 +86,12 @@ Aslref produces the following errors:
 This parse error is coming from a check inside the aslref parser, where equal precedence operators
 are required to be wrapped in parenthesis to disambiguate operation order.
 
+Aslref produces the following errors:
+> FAIL - aslref false | bnfc true: asllib/tests/ASLTypingReference.t/TypingRule.SDecl.bad1.asl
+> FAIL - aslref false | bnfc true: asllib/tests/regressions.t/local_constants.asl
+These errors are due to moving local constants behind the `--allow-local-constants` flag,
```

Comment:
I think/hope that the `internal` attribute in `Parser.mly` makes `menhir2bnfc` aware of these